.travis.yml

```diff
@@ -20,7 +20,7 @@ stages:
 jobs:
   include:
     - stage: validate
-      script: cfn-lint ./templates/**/*
+      script: cfn-lint -i E3012 -- ./templates/**/*
```

Comment:
this will make cfn-lint apply ignore to all files.  I suggest you set ignore to a specific section of the file with a `metada` block.  example here: https://github.com/Sage-Bionetworks/service-catalog-library/blob/master/s3/sc-s3-synapse-ra-v1.0.0.yaml#L72